### PR TITLE
improvement(theme-chalk): visual enhancement

### PR DESCRIPTION
**Pull request by jw-foss**
- Add visual enhancement for keyboard navigation on form items.

### These visual changes will only happen when it comes to keyboard navigation. Mouse interactions will stay intact.

CleanShot 2022-05-08 at 12 24 15@2x

CleanShot 2022-05-08 at 12 25 35@2x

CleanShot 2022-05-08 at 12 26 31@2x

CleanShot 2022-05-08 at 12 26 44@2x

CleanShot 2022-05-08 at 12 39 06@2x

CleanShot 2022-05-08 at 12 39 26@2x

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
I also deleted some unnecessary handlers/attributes in order to prevent them from taking over the browser's functionalities. /cc @Giwayume

File: packages/theme-chalk/src/switch.scss [hunk 1]
```diff
@@ -123,6 +124,13 @@ $switch-button-size: map.merge(
     height: 0;
     opacity: 0;
     margin: 0;
+
+    &:focus-visible {
+      & + .#{$namespace}-switch__core {
```
Comment (Giwayume) [inline packages/theme-chalk/src/switch.scss:129]: Looks like the style isn't working when the switch has `active-text`, this should fix:

`& ~ .#{$namespace}-switch__core {`

**Review by Giwayume — CHANGES_REQUESTED** [review]
Looks very pretty.

File: packages/theme-chalk/src/switch.scss (hunk 1 above)
Comment (JeremyWuuuuu) [reply packages/theme-chalk/src/switch.scss]: Fixed!

**Review by Giwayume — APPROVED** [review]

File: packages/theme-chalk/src/button.scss [hunk 2]
```diff
@@ -166,6 +172,12 @@ $button-icon-span-gap: map.merge(
         background-color: getCssVar('fill-color', 'light');
       }
 
+      &:focus-visible {
+        border-color: transparent;
+        outline: 2px solid getCssVar('button', 'border-color');
```
Comment (YunYouJun) [inline packages/theme-chalk/src/button.scss:177]: Maybe you should use `getCssVar('button', 'active', 'border-color');`

File: packages/theme-chalk/src/button.scss [hunk 3]
```diff
@@ -59,6 +59,12 @@ $button-icon-span-gap: map.merge(
       background-color: getCssVar('button', 'active', 'bg-color');
       outline: none;
     }
+
+    &:focus-visible {
+      border-color: transparent;
+      outline: 2px solid getCssVar('button', 'border-color');
```
Comment (YunYouJun) [inline packages/theme-chalk/src/button.scss:65]: too

**Comment by ryuhangyeong** [comment]
thank you for telling me. The old method is more usable. I will close the current issue. @Giwayume

**Pull request merged**